lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'dish.rb'
+require_relative 'price_calculator.rb'
+
+class Menu
+  attr_reader :dishes, :price_list
+  def initialize(price_calculator_class: Price_calculator)
+    @dishes = []
+    @price_list = Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)